src/ConformalTracking.cc

```diff
@@ -560,6 +560,15 @@ void ConformalTracking::processEvent(LCEvent* evt) {
 
   for (auto const& parameters : _stepParameters) {
     runStep(kdClusters, nearestNeighbours, conformalTracks, collectionClusters, parameters);
+    streamlog_out(DEBUG9) << "STEP " << parameters._step << ": nr tracks = " << conformalTracks.size() << std::endl;
+    for (auto const& confTrack : conformalTracks) {
+      streamlog_out(DEBUG9) << "- Track " << &confTrack << " has " << confTrack->m_clusters.size() << " hits" << std::endl;
+      for (unsigned int ht = 0; ht > confTrack->m_clusters.size(); ht++) {
+        SKDCluster kdhit = confTrack->m_clusters.at(ht);
```

Comment:
`SKDCluster const& kdhit ...` to avoid incrementing and decrementing the reference count of the shared_ptr